libs/ng-utils/src/lib/use-http-image-source/use-http-image-source.pipe.ts

```diff
@@ -52,6 +79,12 @@ export class UseHttpImageSourcePipe implements PipeTransform, OnDestroy {
     return this.latestValue || this.loadingImagePath;
   }
 
+  /**
+   * To perform cleanup before component is destroyed.
```

Comment:
You don't need to document the `ngOnDestroy` method, because it is not ever called by any user